engine/src/gfx/cockpit.cpp

```diff
@@ -2618,9 +1664,9 @@ void GameCockpit::Draw() {
                 headtrans.back().p.j = shake_mag * shakin * cos(1.3731 * theta) * cockpitradial
                         / 100;  //AccessCamera()->GetPosition().j+shakin*cos(theta);
                 headtrans.back().p.k = 0;                 //AccessCamera()->GetPosition().k;
-                headtrans.back().p.i += warp_shake_mag * cos(wtheta) * sqr(warp_strength) * cockpitradial
+                headtrans.back().p.i += warp_shake_mag * cos(wtheta) * sqrtf(warp_strength) * cockpitradial
```

Comment:
Why `sqrtf`? Why not just `sqrt`?

Reply:
It was originally a macro here in the previous MR and I replaced it with a standard function. I usually use std::sqrt in such places. For some reason, the IDE didn't like either that or sqrt and I didn't spend much time on it. 
It's possible the reason for it was that warp_strength is a float. Most variables in VS are floats for reasons of performance. I'm not sure if this is still a valid reason in today's architectures, but converting them en masse is asking for trouble.

Reply:
OK, we can leave these as is then.